rc/core/python.kak

```diff
@@ -47,19 +50,64 @@ add-highlighter -group /python/comment       fill comment
 
     # Highlight keywords
     printf %s "
-        add-highlighter -group /python/code regex '\b(${values})\b' 0:value
-        add-highlighter -group /python/code regex '\b(${meta})\b' 0:meta
-        add-highlighter -group /python/code regex '\b(${keywords})\b' 0:keyword
-        add-highlighter -group /python/code regex '\b(${functions})\b\(' 1:builtin
+        add-highlighter -group /python/code regex '\b(${values})\b' 1:value
+        add-highlighter -group /python/code regex '(?:^|[^.])\b(${meta})\b' 1:meta
+        add-highlighter -group /python/code regex '(?:^|[^.])\b(${keywords})\b' 1:keyword
+        add-highlighter -group /python/code regex '(?:^|[^.])\b(${functions})\b\(' 1:builtin
     "
 
     # Highlight types and attributes
     printf %s "
-        add-highlighter -group /python/code regex '\b(${types})\b' 0:type
+        add-highlighter -group /python/code regex '\b(${types})\b' 1:type
         add-highlighter -group /python/code regex '@[\w_]+\b' 0:attribute
     "
 }
 
+# extended syntax
+
+%sh{ if [ "$kak_opt_python_use_extended_syntax" = "true" ]; then
```

Comment:
I understand the logic behind this, but at this point if there isn't going to be a separate file for extended highlighting I think it's better if everything is merged at once. Having options to control the behaviour of a script at load time really isn't part of the editor's paradigm, as behaviour is mostly controlled by what script is loaded or not.